services/comms-service/src/telemetry.rs

```diff
@@ -24,18 +24,26 @@ use std::sync::{Arc, Mutex};
 pub struct CommsTelemetry {
     /// Errors that have occured within the communication service.
     pub errors: Vec<String>,
+    /// Number of packets uplinked where an error occured.
```

Comment:
I think "Number of bad uplink packets" may sound a bit better.

Reply:
Yes